src/lib/fonts.js

```diff
@@ -1,10 +1,6 @@
-import {
-  Font
-} from '@react-pdf/core'
+import { Font } from '@react-pdf/core'
 
-if (process.env.NODE_ENV !== 'production') {
-  require('dotenv').config()
-}
+require('dotenv').config()
```

Comment:
I'd prefer follow [code snippet](https://github.com/orbiting/proxy/blob/47779db8b146bdcc89d39acf0b54ffa7c59f7f2e/index.js#L8):
```
const DEV = process.env.NODE_ENV && process.env.NODE_ENV !== 'production'
if (DEV || process.env.DOTENV) {
  require('dotenv').config()
}
```

This allows to use the app without installing dev dependencies. And if for example `dotenv` became malicious and started stealing credentials it wouldn't run in prd. To test production locally with a `.env` file you can than use `DOTENV=1 NODE_ENV=production npm start`.

Reply:
Agreed!